### PR TITLE
Create task_4

### PR DESCRIPTION
**Pull request by AnnaSovpel**

### CODE REVIEW

File: task_6/task_4/Task4.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+      Repellat cumque ea harum laudantium rem consequatur quod tenetur, hic at, quasi voluptate dolorem neque quidem modi velit dicta dolore? Rerum optio, ipsa ipsam fugit quibusdam cum velit cumque culpa.
+      Facilis, omnis repellendus nesciunt tenetur dicta culpa magnam id nam iste impedit deserunt iure libero officiis repudiandae at cumque cum odit voluptatem accusantium quo ex, harum qui quis? Nesciunt, repudiandae!
+      Vel incidunt dolorum obcaecati, hic rem quaerat quasi quo officia esse voluptate, autem, nostrum aliquid accusantium? Tempora veritatis impedit, vel tenetur sapiente repudiandae et culpa facilis? Repellat, aspernatur natus! Dicta.
+      Odio commodi eum repellendus dolores velit delectus, esse voluptatibus quam impedit magni quisquam maiores quidem voluptatem minima. At cupiditate sit animi asperiores, eum ipsum enim doloribus provident quam corporis mollitia!
+    </div>
+    <h2 style="text-align:center; font-family: Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif; 
+    margin: 30px;">UPCOMING COURSES</h2>
```
Comment (Killerkip10) [inline task_6/task_4/Task4.html]: Please use css property

Comment (AnnaSovpel) [reply task_6/task_4/Task4.html]: Done.

**Review by Killerkip10 — COMMENTED** [review]
## Homework 4
### Task 1 
20 points
### Task 2
20 points
### Task 3
15 points
### Task 4
- Text doesn't have Roboto font (-5 points);
- Errors in html validator (-5 points);

40 points

### Total 90 points

## Homework 5
### Task 1
- Layout misstake. First row has small height. (-5 points);

20 points
### Task 2
25 points
### Task 3
- 3.6 Incorrect styles when click to the button (-5 points);
- Layout misstake. Buttonts 'Записать' and 'Программа курса' are not located by center (-5 points);
- Errors in html validator (-5 points);

35 points

### Total 80 points

File: start_your_career.html [hunk 2]
```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
```
Comment (Killerkip10) [inline start_your_career.html]: Please, create new folder here and move this file to this folder. Also move background-image file to this folder.

File: start_your_career.html [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <style>
+    .subscribe-button {
+       color: white;    
+       padding: 10px 37px;
+       border-radius: 5px;
+       border: 10px none;
```
Comment (Killerkip10) [inline start_your_career.html]: This border in your case is useless

File: start_your_career.html [hunk 4]
```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html lang="zxx">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <style>
+    .subscribe-button {
+       color: white;
```
Comment (Killerkip10) [inline start_your_career.html]: Please use hex format

File: start_your_career.html [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+    <title>Document</title>
+    <style>
+    .subscribe-button {
+       color: white;    
+       padding: 10px 37px;
+       border-radius: 5px;
+       border: 10px none;
+       margin: 0px 10px;
+       background-color: #a2c744;
+       box-shadow: 0 -3px #769032 inset;
```
Comment (Killerkip10) [inline start_your_career.html]: Will be better to use border-bottom for this

File: start_your_career.html [hunk 6]
```diff
@@ -0,0 +1,155 @@
...
+       cursor: pointer;
+       font-family: Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
```
Comment (Killerkip10) [inline start_your_career.html]: You can set font-family for all page in one space. For example add font-family to body.
I think  font-family: Roboto, sans-serif; is enough.

File: start_your_career.html [hunk 7]
```diff
@@ -0,0 +1,155 @@
...
+    }
+    .subscribe-button:hover {background: #769032;}
+    .subscribe-button:active {background: #a2c744; box-shadow: 0 -3px #769032 inset;}
+   </style>
+   <style>
```
Comment (Killerkip10) [inline start_your_career.html]: Why did you create two style elements?

File: start_your_career.html [hunk 8]
```diff
@@ -0,0 +1,155 @@
...
+   .сourses {
+       color: #2270c8;
+       padding-top: 5px;
```
Comment (Killerkip10) [inline start_your_career.html]: You set padding-top and then set padding: 0px 10px;  At the end padding-top has 0px.

File: start_your_career.html [hunk 9]
```diff
@@ -0,0 +1,155 @@
...
+       padding: 0px 10px;
+       cursor: pointer;
+       font-family: Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
```
Comment (Killerkip10) [inline start_your_career.html]: Remove

File: start_your_career.html [hunk 10]
```diff
@@ -0,0 +1,155 @@
...
+      }
+   .сourses:hover {color: #113c6d;}
+   .сourses:active {color: #2270c8;}
+   </style>
+</head>
+<body style="margin: 0px; background-color: #eceeec;" >
+<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=roboto">
```
Comment (Killerkip10) [inline start_your_career.html]: This element should be located in the head element

File: start_your_career.html [hunk 11]
```diff
@@ -0,0 +1,155 @@
...
+<header style="background-color: #373937; padding: 10px;">
+<div style="line-height: 28px; font-family: Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;">
```
Comment (Killerkip10) [inline start_your_career.html]: I think this div is redundant. You can remove it.

File: start_your_career.html [hunk 12]
```diff
@@ -0,0 +1,155 @@
...
+    <a href="https://www.gomel.it-academy.by/" 
+       style="color: white; text-decoration: none; margin-left: 30px; margin-right: 15px;">List</a>
+    <a href="https://www.gomel.it-academy.by/" 
+       style="color: white; text-decoration: none; margin-right: 15px;">Blog</a>
+    <a href="https://www.gomel.it-academy.by/" 
+       style="color: white; text-decoration: none;">FAQ</a>
+</div>
+</header>
+<main>
+   <div style="background-image: url(https://images.pexels.com/photos/583846/pexels-photo-583846.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=1700);
```
Comment (Killerkip10) [inline start_your_career.html]: You can set background-image to the main element and remove this div

File: start_your_career.html [hunk 13]
```diff
@@ -0,0 +1,155 @@
...
+       style="color: white; text-decoration: none; margin-right: 15px;">Blog</a>
+    <a href="https://www.gomel.it-academy.by/" 
+       style="color: white; text-decoration: none;">FAQ</a>
+</div>
+</header>
+<main>
+   <div style="background-image: url(https://images.pexels.com/photos/583846/pexels-photo-583846.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=1700); 
+               min-height: calc(100vh - 52px); background-position: cover; background-repeat: no-repeat;
+               background-attachment: fixed; display: flex; align-items: center; justify-content: left;">
+    <div style="font-family: Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
```
Comment (Killerkip10) [inline start_your_career.html]: Remove font-family

File: start_your_career.html [hunk 14]
```diff
@@ -0,0 +1,155 @@
...
+                color: white; font-size: 80px; width: 600px; margin-left: 220px; word-spacing: 10px;">
+                Start your career 
+   </div>
+   </div>             
+</main>
+<footer style="margin: 0px 20px;">
```
Comment (Killerkip10) [inline start_your_career.html]: It is not a footer. Use just a div or section

File: start_your_career.html [hunk 15]
```diff
@@ -0,0 +1,155 @@
...
+               min-height: calc(100vh - 52px); background-position: cover; background-repeat: no-repeat;
+               background-attachment: fixed; display: flex; align-items: center; justify-content: left;">
+    <div style="font-family: Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+                color: white; font-size: 80px; width: 600px; margin-left: 220px; word-spacing: 10px;">
+                Start your career 
+   </div>
+   </div>             
+</main>
+<footer style="margin: 0px 20px;"> 
+    <h2 style="text-align:center; font-family: Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;">
```
Comment (Killerkip10) [inline start_your_career.html]: Remove font-family

File: task_6/task_4/Task1.html [hunk 16]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body style="margin: 0;">
+    <div style="border: 4px solid black; margin: 10px; padding: 10px;">
+   <div style="height: 10px; background-color: green; padding: 30px; margin: 30px; color: white;
```
Comment (Killerkip10) [inline task_6/task_4/Task1.html]: You can not set height in this case. Just use padding

File: task_6/task_4/Task1.html [hunk 17]
```diff
@@ -0,0 +1,16 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body style="margin: 0;">
+    <div style="border: 4px solid black; margin: 10px; padding: 10px;">
+   <div style="height: 10px; background-color: green; padding: 30px; margin: 30px; color: white; 
+               line-height: 10px;">Div 1</div>
```
Comment (Killerkip10) [inline task_6/task_4/Task1.html]: And if you use padding you text already will locate by center. Then you can remove line-height

File: task_6/task_5/Task1.html [hunk 18]
```diff
@@ -0,0 +1,35 @@
...
+    <h1 style="text-align: center;">Добыча драгоценных камней</h1>
+    <table border="1" style="border-collapse: collapse; width: 100%;">
+<tr style="background-color: green; text-align: center;">
+    <td></td>
+    <td><strong>2010</strong> </td>
+    <td><strong>2012</strong> </td>
+    <td><strong>2014</strong> </td>
+</tr>
+<tr style="background-color: #ffffd1;">
+    <td style="width: 100px; height: 30px; padding: 10px;">Сапфиры</td>
```
Comment (Killerkip10) [inline task_6/task_5/Task1.html]: heigth is redundant in this case. Remove it

File: task_6/task_5/Task2.html [hunk 19]
```diff
@@ -0,0 +1,50 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body style="margin: 10px 381px;">
+    <table style="border-collapse: collapse; width: 100%;">
+<tr style="border-top: 1px solid #b5cde7; 
+           border-left: 1px solid #b5cde7; border-right: 1px solid #b5cde7; color: #3966b3;">
+    <td style="width: 100px; padding: 10px;"> <strong>Company</strong></td>
+    <td style="text-align: center;"> <strong>Q1</strong></td>
```
Comment (Killerkip10) [inline task_6/task_5/Task2.html]: Please use css propertyes instead of strong elements

**Review by Killerkip10 — COMMENTED** [review]
Please resolve all this comments

**Deadline 31.07.2020 18.30** or **-20** points

File: start_your_career.html (hunk 3 above)
Comment (AnnaSovpel) [reply start_your_career.html]: Done.

File: start_your_career.html (hunk 4 above)
Comment (AnnaSovpel) [reply start_your_career.html]: Done

File: start_your_career.html (hunk 6 above)
Comment (AnnaSovpel) [reply start_your_career.html]: Done

File: start_your_career.html (hunk 15 above)
Comment (AnnaSovpel) [reply start_your_career.html]: Done

File: task_6/task_4/Task1.html (hunk 16 above)
Comment (AnnaSovpel) [reply task_6/task_4/Task1.html]: Done

File: task_6/task_4/Task1.html (hunk 17 above)
Comment (AnnaSovpel) [reply task_6/task_4/Task1.html]: Done

File: task_6/task_5/Task2.html (hunk 19 above)
Comment (AnnaSovpel) [reply task_6/task_5/Task2.html]: Done

File: start_your_career.html (hunk 7 above)
Comment (AnnaSovpel) [reply start_your_career.html]: Done

File: start_your_career.html (hunk 9 above)
Comment (AnnaSovpel) [reply start_your_career.html]: Done

File: start_your_career.html (hunk 11 above)
Comment (AnnaSovpel) [reply start_your_career.html]: Done

File: start_your_career.html (hunk 13 above)
Comment (AnnaSovpel) [reply start_your_career.html]: Done

File: task_6/task_5/Task1.html (hunk 18 above)
Comment (AnnaSovpel) [reply task_6/task_5/Task1.html]: Done

File: start_your_career.html (hunk 8 above)
Comment (AnnaSovpel) [reply start_your_career.html]: Done

File: start_your_career.html (hunk 12 above)
Comment (AnnaSovpel) [reply start_your_career.html]: Done

File: start_your_career.html (hunk 14 above)
Comment (AnnaSovpel) [reply start_your_career.html]: Done

File: start_your_career.html (hunk 10 above)
Comment (AnnaSovpel) [reply start_your_career.html]: Done

File: start_your_career.html (hunk 5 above)
Comment (AnnaSovpel) [reply start_your_career.html]: Done, but I used a different method to execute Your comment.

File: start_your_career.html (hunk 11 above)
Comment (Killerkip10) [reply start_your_career.html]: You did't remove it))

File: start_your_career.html (hunk 14 above)
Comment (Killerkip10) [reply start_your_career.html]: Again, you didn't remove it

File: task_6/task_5/Task2.html (hunk 19 above)
Comment (Killerkip10) [reply task_6/task_5/Task2.html:13]: You still use <strong> everywhere

File: start_your_career.html (hunk 5 above)
Comment (Killerkip10) [reply start_your_career.html]: box-shadow already exists

Comment (AnnaSovpel) [reply start_your_career.html]: Done

File: start_your_career.html (hunk 11 above)
Comment (AnnaSovpel) [reply start_your_career.html]: Done

File: start_your_career.html (hunk 14 above)
Comment (AnnaSovpel) [reply start_your_career.html]: Done

File: task_6/task_5/Task2.html (hunk 19 above)
Comment (AnnaSovpel) [reply task_6/task_5/Task2.html]: Done

File: start_your_career.html (hunk 14 above)
Comment (Killerkip10) [reply start_your_career.html]: no no no)))

Comment (AnnaSovpel) [reply start_your_career.html]: Finally done!)

**Pull request merged**